lib/svgo.test.js

```diff
@@ -285,13 +285,13 @@ test('provides informative error in result', () => {
 });
 
 test('provides code snippet in rendered error', () => {
+  expect.assertions(1);
   const svg = `<svg viewBox="0 0 120 120">
   <circle fill="#ff0000" cx=60.444444" cy="60" r="50"/>
 </svg>
 `;
   try {
     optimize(svg, { path: 'test.svg' });
-    expect(true).toEqual(false);
   } catch (error) {
     expect(error.toString())
       .toEqual(`SvgoParserError: test.svg:2:29: Unquoted attribute value
```

Comment:
why not convert this one to using `toThrow` too?

Reply:
The `.toThrow()`  matcher matches against the `.message` attribute only, not to `toString()`. Here `toString()` is being is used to assert the source and line numbers in addition to the message.